web/src/utilities/airgapUploader.js

```diff
@@ -1,3 +1,5 @@
+// TODO: fix linting issues
+/* eslint-disable */
```

Comment:
opted not to manually fix anything in this pr. I did a blanket ignore on files that had several linting errors.